.github/workflows/storybook-chromatic.yml

```diff
@@ -0,0 +1,35 @@
+name: QA
+
+on: [pull_request]
+
+jobs:
+  chromatic-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-qa-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-qa-${{ env.cache-name }}-
+            ${{ runner.os }}-qa-
+            ${{ runner.os }}-
+
+      - name: Install deps
+        run: npm ci
+
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
```

Comment:
Freeze the 3rd party:

```suggestion
        uses: chromaui/action@a89b674adf766dbde41ad9ea2b2b60b91188a0f0 # v6.17.4
```

Reply:
I'm wondering if the same shouldn't be applied to dashboard npm dependencies. As we are using `^` and by `npm install` we catch all patch versions e.g
![CleanShot 2023-05-09 at 10 52 42@2x](https://user-images.githubusercontent.com/9116238/237045608-4d560dff-8133-494b-8507-18db23271150.jpg)